### PR TITLE
accessibilty-fixes

### PR DESCRIPTION
**Pull request by muneeb3123**
This PR Includes:
 -**Page Title**: No issues found
**Alt images**:Found some images Alt needs a more detailed description 
**Headings**: Some headings were not following the hierarchy
**Contrast**:Requires improvement however is according to Figma design
**Resize Texts**: No issues found Text resizes properly 
**Keyboard access and visual focus**:Got rid of tab-index greater than 0
**Interaction**: There are no labels but they are optional Moving, Flashing, or Blinking Content: No issues found              
  **Basic structure check**: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h1 id="Headline-Title">I’m Muneeb<br>
             some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate
             to contact me. </p>
         <div class="socialmedia">
-            <h3 class="helpingtext">LET'S CONNECT</h3>
+            <h2 class="helpingtext">LET'S CONNECT</h2>
```
Comment (EddxSotz) [inline index.html:39]: Changed to an h2 instead to adhere to correct headings practices

File: index.html [hunk 2]
```diff
@@ -62,7 +62,7 @@ <h3 class="helpingtext">LET'S CONNECT</h3>
 
     <section class="Grid-boxes">
         <article class="Box">
-            <img class="box-image" src="req/tonic.png" alt="tonic">
+            <img class="box-image" src="req/tonic.png" alt="protfolio image tonic">
```
Comment (devalibello) [inline index.html:65]: The alt attribute was modified to be more descriptive

File: index.html [hunk 3]
```diff
@@ -83,7 +83,7 @@ <h2 class="content-title">Tonic</h2>
             </div>
         </article>
         <article class="Box">
-            <img class="box-image" src="req/multi.png" alt="multi">
+            <img class="box-image" src="req/multi.png" alt="protfolio image multi">
```
Comment (devalibello) [inline index.html:86]: The alt attribute was modified to be more descriptive

File: index.html [hunk 4]
```diff
@@ -104,7 +104,7 @@ <h2 class="content-title">Multi-Post Stories</h2>
             </div>
         </article>
         <article class="Box">
-            <img class="box-image" src="req/tonic_2.png" alt="tonic_2">
+            <img class="box-image" src="req/tonic_2.png" alt=" protfolio image tonic_2">
```
Comment (devalibello) [inline index.html:107]: The alt attribute was modified to be more descriptive

File: index.html [hunk 5]
```diff
@@ -125,7 +125,7 @@ <h2 class="content-title">Tonic</h2>
             </div>
         </article>
         <article class="Box">
-            <img class="box-image" src="req/multi_2.png" alt="multi_2">
+            <img class="box-image" src="req/multi_2.png" alt=" protfolio image multi_2">
```
Comment (devalibello) [inline index.html:128]: The alt attribute was modified to be more descriptive

File: index.html [hunk 6]
```diff
@@ -146,7 +146,7 @@ <h2 class="content-title">Multi-Post Stories</h2>
             </div>
         </article>
     </section>
-    <section id="about-section" tabindex="20">
```
Comment (devalibello) [inline index.html:149]: The "tabindex" attribute was removed to enable users to navigate through the structure of the website using the tab key. without hitch-free

File: index.html (hunk 1 above)
Comment (hiromataba) [reply index.html:39]: - good practice and comment 😉

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @muneeb3123 and @EddxSotz    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**